.gitignore

```diff
@@ -48,6 +48,7 @@ yarn-error.log*
 
 # devcontainer files
 !.devcontainer/chefs_local/*.sample
+!.devcontainer/chefs_local/*.sample.json
```

Comment:
I think the ignore for `local.*` on line 22 will exclude the new file?

Reply:
Cleaned up gitignore after removing keycloak realm file.